python/cudf_polars/cudf_polars/experimental/dask_serialize.py

```diff
@@ -57,3 +63,11 @@ def _(header, frames) -> DataFrame:
             # Copy the second frame (the gpudata in host memory) back to the gpu
             frames = frames[0], plc.gpumemoryview(rmm.DeviceBuffer.to_device(frames[1]))
             return DataFrame.deserialize(header, frames)
+
+    @dask_deserialize.register(Column)
```

Comment:
Most of this `Column` logic is identical to the `DataFrame` logic. However, I don't think we gain much by avoiding the separation. Also, it's not immediately obvious to me *how* we can know the type within the dispatch function (unless we pickle/unpickle the type in the header - which we should definitely avoid).